### ISSUE TITLE
Querstion about cache invalidation

### ISSUE THREAD

**Issue by pasztorl** [IBM/varnish-operator#113]
Hi, If I run multiple varnish instances is there a vay to purge the caches so the purge command is distributed between the varnish pods?

**Comment by cin** [comment]
Unfortunately, there is not a built in way to do this with the varnish operator. Are you wanting to run the purge command across all varnish instances at once or serially? If the former, perhaps [[kubectl-parallel-exec](https://github.com/cin/kubectl-parallel-exec)](https://github.com/cin/kubectl-parallel-exec) could help. If this is a feature you'd like added, feel free to contribute a PR. I have been maintaining a fork of this repo [here](https://github.com/cin/varnish-operator). I need to do some maintenance on this project in general.

**Comment by pasztorl** [comment]
Yes, in general I would like to control the cache with a single request, like on this project: https://github.com/mittwald/kube-httpcache
